Bugzilla/API/V1/Github.pm

```diff
@@ -164,7 +171,167 @@ sub pull_request {
   return $self->render(json => {error => 0, id => $attachment->id});
 }
 
-sub verify_signature {
+sub push_comment {
+  my ($self) = @_;
+  my $template = Bugzilla->template;
+  Bugzilla->usage_mode(USAGE_MODE_MOJO_REST);
+
+  # Return early if push commenting is not allowed
+  return $self->code_error('github_push_comment_disabled')
+    if !Bugzilla->params->{github_push_comment_enabled};
+
+  # Return early if not a push or ping event
```

Comment:
Not sure how this would work since they caller still uses the full endpoint even for pings (i.e. `/rest/github/{push_comment,pull_request}`). This only happens once that I know of when the admin first sets up the webhook in Github. The amount of code that is duplicated is so small I am also reluctant to create a separate ping function to call from both functions. Ideas?